debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -709,9 +709,24 @@ public static LogMiningDmlParser parse(String value, String defaultValue) {
      */
     private static class SystemTablesPredicate implements TableFilter {
 
+        private final Configuration config;
+
+        SystemTablesPredicate(Configuration config) {
+            this.config = config;
+        }
+
         @Override
         public boolean isIncluded(TableId t) {
-            return !OracleConnectorConfig.getExcludedSchemaNames().contains(t.schema().toLowerCase());
+            return !isExcludedSchema(t) && !isFlushTable(t);
+        }
+
+        private boolean isExcludedSchema(TableId id) {
+            return OracleConnectorConfig.getExcludedSchemaNames().contains(id.schema().toLowerCase());
```

Comment:
Let's make the list backing `getExcludedSchemaNames()` a constant instead of the local `Arrays.asList()` call.